cypress/integration/headerSpec.js

```diff
@@ -0,0 +1,39 @@
+import {
+  getElement,
+  shouldContainText,
+  shouldContainStyles,
+} from '../support/testHelper';
+
+describe('Header Tests', () => {
+  // eslint-disable-next-line no-undef
+  before(() => {
+    cy.visit('/article/scenario-25');
+  });
+
+  it('should render the BBC News branding', () => {
+    const headerElement = getElement('header');
+    shouldContainStyles(headerElement, 'height', '80px');
+    shouldContainStyles(headerElement, 'background-color', 'rgb(184, 0, 0)');
+
+    const anchorElement = getElement('header a');
+    shouldContainText(anchorElement, 'BBC News');
+
+    anchorElement.focus();
```

Comment:
Can you please add another `it` block with `it('should have a focus state'` to surround this assertion?